Helios/Helios.csproj

```diff
@@ -29,8 +29,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\EosSharp.1.1.5488.21795\lib\net45\EosSharp.dll</HintPath>
     </Reference>
-    <Reference Include="HidSharp, Version=2.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\HidSharp.2.1.0\lib\net35\HidSharp.dll</HintPath>
+    <Reference Include="HidSharp, Version=2.1.0.0, Culture=neutral, PublicKeyToken=0867af7205b1c8cf, processorArchitecture=MSIL">
+      <HintPath>..\packages\StreamDeckSharp.3.0.0\lib\netstandard2.0\HidSharp.dll</HintPath>
```

Comment:
why is this change ok?   I don't know enough about .NET standard stuff to understand why this is ok?    I always keep my Framework vs Standard things separate out of fear, I guess...   Do the other customers of HidSharp still work correctly after this change?

Reply:
oh I see, my mistake.... I didn't have coffee yet and thought Core when it said Standard :).   Never mind, carry on and ignore.

Reply:
Those packages were added using NuGet. I wonder why it defaulted to netstandard 2.0... I'll look into it and see why that changed like that...